proto/regen/group/v1alpha1/events.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package regen.group.v1alpha1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/group";
+
+// EventCreateGroup is an event emitted when a group is created.
+message EventCreateGroup {
+
+  // group_id is the unique ID of the group.
+  string group_id = 1;
+
+  // admin is the admin address of the group.
+  string admin = 2;
+}
+
+// EventUpdateGroup is an event emitted when a group is updated.
+message EventUpdateGroup {
+
+  // group_id is the unique ID of the group.
+  string group_id = 1;
+
+  // admin is the admin address of the group.
+  string admin = 2;
+}
+
+// EventCreateGroupAccount is an event emitted when a group account is created.
+message EventCreateGroupAccount {
+
+  // group_account is the address of the group account.
+  string group_account = 1;
+
+  // admin is the admin address of the group.
+  string admin = 2;
+}
+
+// EventUpdateGroupAccount is an event emitted when a group account is updated.
+message EventUpdateGroupAccount {
+
+  // group_account is the address of the group account.
+  string group_account = 1;
+
+  // admin is the admin address of the group.
+  string admin = 2;
+}
```

Comment:
What is the purpose in only including `admin` for each of these events? If we wanted to be minimalist we would only include `group_id` and `group_account` I think. If we wanted more info, would we maybe reference each member??

Also, events are missing for proposals and voting. I'm fine if that happens in separate PRs, but we should have it.

Reply:
I guess it was to replicate the existing events https://github.com/regen-network/cosmos-modules/blob/master/incubator/group/group_handler.go#L105-L108 which included the signer but I agree we can just remove it for now to keep it minimalist.
If we wanted more info, we could just add the corresponding request data (eg comment, members...).

Also, as mentioned in the PR description above, right now we're are using different endpoints for updating a group whether we wanna update its members, admin or comments. It would be better UX-wise to just have one update endpoint if possible and that would corresponds better to the current events, wdyt?

I added a new issue to add more events: https://github.com/regen-network/regen-ledger/issues/215

Reply:
Could you explain a little more what you mean by one update endpoint?

Reply:
If you look at https://github.com/regen-network/regen-ledger/blob/marie/136-x-group/proto/regen/group/v1alpha1/tx.proto#L18-L25, you can see that we have multiple methods for updating groups. I thought it would be better to have just one method `UpdateGroup` instead of `UpdateGroupMembers`, `UpdateGroupAdmin`, etc.